framework/doc/content/source/auxkernels/BuildArrayVariableAux.md

```diff
@@ -0,0 +1,20 @@
+# BuildArrayVariableAux
+
+## Description
+
+`BuildArrayVariableAux` will fill the values of an array variable using a
+standard variable for each of the array components.
```

Comment:
```suggestion
!syntax description /AuxKernels/BuildArrayVariableAux
```

You should be doing this and then natively getting the description from the paramaters (probably a good idea to move this content to the `addClassDescription()`, it's a little more descriptive.

Reply:
Nice, yeah this seems nice to only have one location for the canonical description of this class.